vault/resource_identity_entity_test.go

```diff
@@ -188,5 +238,25 @@ resource "vault_identity_entity" "entity" {
   metadata = {
     version = "2"
   }
+  disabled = true
```

Comment:
Might want to check this parameter in `TestAccIdentityEntityUpdate()` now.

Reply:
Checked!